tests/ihontaryk/homework_10/test_recreate_generators.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from ihontaryk.homework_10.recreate_generators import (custom_zip, custom_enumerate,
+                                                       custom_filter, custom_map, filter_int)
+
+
+@pytest.mark.parametrize('test_iterable1, test_iterable2, test_iterable3, expected_result',
+                         [(["a", "b", "c"], ["x", "y", "z"], [1, 2, 4],
+                           [("a", "x", 1), ("b", "y", 2), ("c", "z", 4)]),
+                          (["a", "b", "c"], [0, 3, 6], ["x", "y", "z"],
+                           [("a", 0, "x"), ("b", 3, "y"), ("c", 6, "z")])
+                          ])
```

Comment:
does 2nd test introduce any specific case, which you try to check, is answer is no, remove it

Reply:
changed testing data for the 2nd test